### ISSUE TITLE
java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.collections.ArraysKt___ArraysKt.contains, parameter $this$contains

### ISSUE THREAD

**Issue by duncte123** [Almighty-Alpaca/JetBrains-Discord-Integration#92]
Got the following exception (discord was not running on the computer in question)

Plugin version: 1.3.0
IDE version: PhpStorm 2020.1.1
Build #PS-201.7223.96, built on April 30, 2020.
Runtime version: 11.0.6+8-b765.40 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.3.0-51-generic
GC: ParNew, ConcurrentMarkSweep
Registry: run.processes.with.pty=TRUE, debugger.watches.in.variables=false
Non-Bundled Plugins: com.almightyalpaca.intellij.plugins.discord, com.tabnine.TabNine, izhangzhihao.rainbow.brackets, ru.adelf.idea.dotenv, de.espend.idea.laravel
Current Desktop: ubuntu:GNOME

```
java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.collections.ArraysKt___ArraysKt.contains, parameter $this$contains
	at kotlin.collections.ArraysKt___ArraysKt.contains(_Arrays.kt)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseService$readPlugins$matches$4.invoke(DiagnoseService.kt:153)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseService$readPlugins$matches$4.invoke(DiagnoseService.kt:40)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseServiceKt$sam$java_util_function_Predicate$0.test(DiagnoseService.kt)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.ReduceOps$5.evaluateSequential(ReduceOps.java:257)
	at java.base/java.util.stream.ReduceOps$5.evaluateSequential(ReduceOps.java:248)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.count(ReferencePipeline.java:605)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseService.readPlugins(DiagnoseService.kt:154)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseService.access$readPlugins(DiagnoseService.kt:40)
	at com.almightyalpaca.jetbrains.plugins.discord.plugin.diagnose.DiagnoseService$plugins$1.invokeSuspend(DiagnoseService.kt:44)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

**Comment by InfRandomness** [comment]
Could you share the steps to reproduce the issue (if you know how to reproduce it) ?

**Comment by duncte123** [comment]
> Could you share the steps to reproduce the issue (if you know how to reproduce it) ?

How to reproduce is to not have discord running while the plugin is enabled

**Comment by InfRandomness** [comment]
A fix has been released 12h ago on the repository.

**Comment by Almighty-Alpaca** [comment]
A fix for this was released this morning with version v1.4.0
